### ISSUE TITLE
How to use the lastest UI5 Web Components version via this Angular Package?

### ISSUE THREAD

**Issue by MichaelFlucher** [SAP/ui5-webcomponents-ngx#153]
### **Feature Request Description**

More or less general question: Is there a process or predefined way, to directly use the newest release of Web Components (currently 2.3.0)? Currently, even with `0.3.0-rc.15` I have to stick to `2.1.1`, if I don't want to use `npm install --force`. So is there a plan to be more up-to-date or to circumvent this problem?

### **Proposed Solution**
Totally not sure, since it's not an easy topic with many different dependencies.

### **Proposed Alternatives**
Multiple RCs with different version dependencies, even if untested?

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

A clear and concise description of the impact/urgency of the required feature.

### **Stakeholder Info** (if applicable)

- Organization: {...}
- Business impact: {...}

**Comment by droshev** [comment]
@MichaelFlucher yes, it should be a straight forward step. Let me follow up with that.

**Comment by droshev** [comment]
the version 0.3.2-rc.0 includes the latest (2.4.0 version of ui5-webcomponents)
